docs/src/dataDisplay/personCoreMilestones/examples/constants.js

```diff
@@ -1,6 +1,10 @@
 export const DATA_ADULT_MALE = {
     acceptedChristDate: '2018-01-01T00:00:00',
     activeInMissionsDate: null,
+    activate1Date: '2020-02-23T00:00:00',
+    activate2Date: null,
```

Comment:
Why is `activate2Date` `null` but `hasAttendedActivate2` is `true` (Line 20)?
That seems incorrect.

Reply:
Fixed: https://github.com/saddlebackdev/react-cm-ui/pull/496/commits/e5f61c48bcb69779107193319f469bc7cb14973b#diff-f60c55356e4158367c963a1656a1713cbfc285ebd93c4008ed382f5452c7975aL20-R20